_pages/how-we-work/contractors.md

```diff
@@ -28,6 +28,12 @@ In general, contractors "embedded" in TTS (working on TTS projects most or full-
 
 The [GSA Access Card ("badge", "PIV card")]({{site.baseurl}}/piv/) is part of the contractor's onboarding process. The sensitivity level of their work (i.e. manipulates data or system configurations, access to PII) and position will determine if a GSA Access Card is required.
 
+## Equipment
```

Comment:
Can we merge the section below into this?

Reply:
Didn't even see that, good catch -- yes, the latest commit merges them.